tests/smoke_test.go

```diff
@@ -269,9 +273,7 @@ func TestPackageGetSchema(t *testing.T) {
 
 //nolint:paralleltest // disabled parallel because we change the plugins cache
 func TestPackageGetMapping(t *testing.T) {
-	if os.Getenv("CI") != "" {
-		t.Skip("Skipping because it gets rate limited in CI")
-	}
```

Comment:
So we're no longer disabling these tests in CI, but also enabling the automatic plugin acquisition.  Did we determine that these tests weren't actually at fault for using too many API calls then?

Reply:
>  Did we determine that these tests weren't actually at fault for using too many API calls then?

Correct. These tests do automatic plugin downloads but they download like 2 plugins. What was the issue was a load of other tests (mostly in pkg/engine/lifecycle) we're doing _loads_ of requests to github to try to download fake providers (like "pkgA"). By the time we got to these tests we'd burnt our quota on failed downloads from the other tests.